x-pack/plugins/osquery/server/lib/saved_query/saved_object_mappings.ts

```diff
@@ -67,6 +67,9 @@ export const savedQuerySavedObjectMappings: SavedObjectsType['mappings'] = {
     interval: {
       type: 'keyword',
     },
+    timeout: {
+      type: 'short',
+    },
```

Comment:
You will need to add new model versions reflecting the mapping changes to the 3 SO types you updated the mappings of, otherwise existing serverless projects won't be able to pick up the mapping changes. 

You can refer to the [SO modelVersion documentation](https://github.com/elastic/kibana/blob/9b59e84d3e3601a12fa5ff33814393b8f7644010/packages/core/saved-objects/core-saved-objects-server/docs/model_versions.md)